### PR TITLE
Wheelchair accessible labeling

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
Provides invisible labeling for wheelchair accessibility badges so that users with AT can identify whether a leg or itinerary is wheelchair accessible.

### CODE REVIEW

File: packages/itinerary-body/i18n/en-US.yml [hunk 1]
```diff
@@ -81,6 +81,13 @@ otpUi:
       car: by car
       escooter: by e-scooter
       walk: by walking
+    tripAccessibility:
+      itineraryAccessibility: "Wheelchair accessibility of this trip: "
+      legAccessibility: "Wheelchair accessibility of this trip leg: "
+      tripAccessibilityUnclear: unknown
+      tripIsAccessible: wheelchair accessible
+      tripIsInaccessible: wheelchair inaccessible
+      tripIsLikelyAccessible: likely wheelchair accessible
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply packages/itinerary-body/i18n/en-US.yml:90]: I would remove "wheelchair" from these four qualifiers (redundant with the label).

File: packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx [hunk 2]
```diff
@@ -75,15 +96,29 @@ const AccessibilityRating = ({
...
+
+  const accessibilityLabel = accessibilityPreface + accessibilityScore;
+
   return (
     <Wrapper
       border={grayscale}
       color={grayscale ? "transparent" : mapped.color}
       large={large}
-      title={mapped.text}
+      leg
```
Comment (binh-dam-ibigroup) [inline packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx:114]: This prop appears to be unused and can be removed.

File: packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx [hunk 3]
```diff
@@ -2,10 +2,13 @@ import { GradationMap } from "@opentripplanner/types";
 import React, { ReactElement } from "react";
 import styled from "styled-components";
 import { Wheelchair } from "@styled-icons/foundation/Wheelchair";
+import { useIntl } from "react-intl";
+import { InvisibleAdditionalDetails } from "../styled";
 
 interface WrapperProps {
   border: boolean;
   color: string;
+  leg: boolean;
```
Comment (binh-dam-ibigroup) [inline packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx:11]: This prop appears to be unused and can be removed.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
What do you think about removing redundant "wheelchair" wording?

File: packages/itinerary-body/i18n/en-US.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/itinerary-body/i18n/en-US.yml]: let me know what you think about [4c78dd2](https://github.com/opentripplanner/otp-ui/pull/675/commits/4c78dd2441ec26478ff5b54bf50a740a9c308d0c)!

File: packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx (hunk 3 above)
Comment (amy-corson-ibigroup) [reply packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx]: Good catch thank you! [cd5f97e](https://github.com/opentripplanner/otp-ui/pull/675/commits/cd5f97eab5b5b479276c3dac8ef3d5721bfb4252)

**Review by binh-dam-ibigroup — COMMENTED** [review]
See my responses.

File: packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx [hunk 4]
```diff
@@ -40,6 +42,7 @@ interface Props {
   gradationMap?: GradationMap;
   grayscale?: boolean;
   large?: boolean;
+  leg?: boolean;
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx]: I went with `isLeg` does that work?

**Review by binh-dam-ibigroup — APPROVED** [review]

**Comment by binh-dam-ibigroup** [comment]
@amy-corson-ibigroup Please update the snapshots.

File: packages/itinerary-body/i18n/en-US.yml [hunk 5]
```diff
@@ -81,6 +81,13 @@ otpUi:
       car: by car
       escooter: by e-scooter
       walk: by walking
+    tripAccessibility:
+      accessible: accessible
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/itinerary-body/i18n/en-US.yml]: Good point, it's not being used so I took it out, we can add it again if the default gradation map changes.

**Review by AdrianaCeric — APPROVED** [review]
Thanks for the change!

**Comment by amy-corson-ibigroup** [comment]
Closed in favor of https://github.com/opentripplanner/otp-ui/pull/682

**Pull request closed without merging**